v22.1/aws-dms.md

```diff
@@ -0,0 +1,140 @@
+
+### Step 2.2. Task settings
+
+1. For the **Editing mode** radio button, keep **Wizard** selected.
+1. For the **Target table preparation mode**, select either **Truncate** or **Do nothing**. All other settings can remain as-is.
+    <img src="{{ 'images/v22.1/aws-dms-task-settings.png' | relative_url }}" alt="AWS-DMS-Task-Settings" style="max-width:100%" />
+
+In order to use **Drop tables on target**, you must [create a new `dms` user](#create-a-new-dms-user) and ensure [`BatchApplyEnabled`](#batchapplyenabled) is set to `False`.
+
+### Step 2.3. Table mappings
+
+1. For the **Editing mode** radio button, keep **Wizard** selected.
+1. Select **Add new selection rule**.
+1. Supply the appropriate **Schema** (if applicable), **Source name** (database name), **Table name**, and **Action**.
+    <img src="{{ 'images/v22.1/aws-dms-table-mappings.png' | relative_url }}" alt="AWS-DMS-Table-Mappings" style="max-width:100%" />
+
+## Step 3. Verify the migration
+
+Data should now be moving from source to target. You can analyze the **Table Statistics** page for information about replication.
+
+1. In **AWS DMS**, open **Database migration tasks** in the sidebar.
+1. Select the task you created in Step 2.
+1. Select **Table statistics** below the **Summary** section.
+
+If your migration failed for some reason, you can check the checkbox next to the table(s) you wish to re-migrate and select **Reload table data**.
+
+## Optional configurations
+
+### Create a new `dms` user
+
+Creating a `dms` user is necessary when using **Drop tables on target** as a target table preparation mode.
+
+1. [Connect to your {{ site.data.products.serverless }} cluster](../cockroachcloud/connect-to-a-serverless-cluster.html).
+
+1. In the SQL client, create a new `dms` admin user to handle the migration. Replace `'<password>'` with a strong password:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    > CREATE USER dms WITH PASSWORD '<password>';
+    > GRANT admin TO dms;
+    > ALTER USER dms SET expect_and_ignore_not_visible_columns_in_copy = true;
+    ~~~
+
+    {{site.data.alerts.callout_danger}}
+    Do not use this user for normal SQL activity. The `expect_and_ignore_not_visible_columns_in_copy` session variable may make it behave unpredictably for normal usage.
+    {{site.data.alerts.end}}
+
+### CloudWatch logs
+
+You can Enable CloudWatch logs for extra insight about the replication. To enable CloudWatch logs:
+
+1. Edit the existing replication task.
+1. Under **Task settings**, select **Enable CloudWatch logs**. From here, you can specify logging levels for each event type:
+    <img src="{{ 'images/v22.1/aws-dms-cloudwatch-logs.png' | relative_url }}" alt="AWS-DMS-CloudWatch-Logs" style="max-width:100%" />
+
+### `BatchApplyEnabled`
+
+The `BatchApplyEnabled` setting can improve replication performance and is recommended when using a target table preparation mode of **Truncate** or **Do nothing**.
```

Comment:
i would say it is "recommended for larger workloads."

Then "The target table preparation mode must be **Truncate** or **Do nothing**."

Reply:
Done.